src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -1857,10 +1857,22 @@ protected override void BeginProcessing()
             if (ArgumentList != null)
             {
                 StringBuilder sb = new StringBuilder();
-                foreach (string str in ArgumentList)
+                foreach (string argument in ArgumentList)
                 {
-                    sb.Append(str);
-                    sb.Append(' ');
+                    if (argument != null && argument.Contains(" "))
```

Comment:
I am wondering if this condition needs to be tweaked to also add the clause `&& !argument.StartsWith("\"")` because the [docs](https://docs.microsoft.com/en-us/dotnet/api/system.diagnostics.processstartinfo.arguments?view=netcore-2.0#System_Diagnostics_ProcessStartInfo_Arguments) say that 2 double quotes are `interpreted as if the quotes weren't there` and `Triple-escape quotation marks to include the character in the final argument received`. A user does not know that low level detail/quirks of `ProcessStartInfo.Arguments` and it would reduce the number of breaking changes.